ChatApp/View/RecipientMessageView.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  MessageView.swift
+//  ChatApp
+//
+//  Created by Ana Dzebniauri on 21.04.23.
+//
+
+import UIKit
+
+class RecipientMessageView: UIView {
+    
+    //MARK: - Properties
+    private let capsuleView = RecipientCapsuleBubble().forAutoLayout()
+    private let mediumBubble = RecipientBubble().forAutoLayout()
+    private let smallBubble = RecipientBubble().forAutoLayout()
```

Comment:
from naming it is not clear is it UIView or some other type's of component. mediumBubbleView  / smallBubbleView would be better